src/couch_replicator/src/couch_replicator.erl

```diff
@@ -182,6 +115,8 @@ rep_result_listener(RepId) ->
         end).
 
 
+-spec wait_for_result(rep_id()) ->
+    {ok, any()} | {error, any()}.
```

Comment:
I wounder if you have a dialyzer warning here. It looks like it should be `{ok, {[_]}} |
{error, any()}`. Feel free to ignore if there is no warning.